docs/source/03_tutorial/06_visualise_pipeline.md

```diff
@@ -96,3 +96,115 @@ kedro viz --load-file my_shareable_pipeline.json
 ```
 
 And this will visualise the pipeline visualisation saved as `my_shareable_pipeline.json`.
+
+
+## Visualise Plotly charts in Kedro-Viz
+
+[Plotly](https://plotly.com/python/) is a free and open source Python library that allows you to make interactive, publication-quality graphs. With the Plotly integration on Kedro-Viz, you can output your interactive charts as part of your pipeline visualisation.
+
+Kedro-Viz aims to help users communicate different aspects of their workflow through an interactive flowchart. With the Plotly integration, we take one step further in this direction to allow our users to effectively share their data insights while exploring the pipeline.
+
+We have also used the Plotly integration to allow users to [visualise metrics from experiments](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html?highlight=experiment%20tracking).
+
+
+```eval_rst
+  .. note:: Kedro's Plotly integration only supports `Plotly Express <https://plotly.com/python/plotly-express/>` charts. 
+```
+
+You can view Plotly charts in Kedro-Viz when you use Kedro's plotly datasets. To enable Plotly you need to update requirements.txt in your kedro project and add either or both of the below datasets. 
+
+ `kedro[plotly.PlotlyDataSet, plotly.JSONDataSet]==0.xx.x`
```

Comment:
I think you can add the version number here because we update this number when we go through the documentation.